test/replication/suite.ini

```diff
@@ -10,7 +10,7 @@ use_unix_sockets_iproto = True
 long_run = prune.test.lua
 is_parallel = True
 fragile = {
-    "retries": 10,
+    "retries": 0,
```

Comment:
Please, don't waste time on stupid changes like this and like outdated comments. This is a very dirty patch which I will heavily rework.

Reply:
> This is a very dirty patch which I will heavily rework.

I understand. This took me 1 second and I simply pointed this out just in case you miss it during refactoring.
I won't point at such things any more if you don't want me to